XTS/localjunit/crash-recovery-tests/src/main/java/com/arjuna/qa/extension/BaseServerKillProcessor.java

```diff
@@ -122,7 +122,9 @@ protected void dumpProcesses(Container container) throws Exception {
         String scriptName = config.get("scriptName");
 
         String dir = "target" + File.separator + "surefire-reports" + File.separator + "processes";
-        runShellCommand("mkdir " + dir);
+        if(!new File(dir).exists()) {
+            runShellCommand("mkdir " + dir);
```

Comment:
I think there is no purpose to throw an exception.

This part only tries to create directory. If fails then prints a warning to log. If pass ok. Next test does the same. It tries to create the directory and if fails it just prints warning etc. The subsequent commands then put data to that directory which is expected to exist.

This change only ensures that there won't be attempt to create a new directory if it exists. It means the subsequent code still uses the same directory, the directory is already existing but there won't be obtrusive warning in test log.